hardware_interface/src/resource_manager.cpp

```diff
@@ -37,6 +37,7 @@
 #include "lifecycle_msgs/msg/state.hpp"
 #include "pluginlib/class_loader.hpp"
 #include "rcutils/logging_macros.h"
+#include "std_msgs/msg/header.h"
```

Comment:
When using Realtime publisher, this should become obsolete. Check any controller code.

Reply:
Why not?